docs/preview/02-Features/02-assertion.md

```diff
@@ -54,6 +49,18 @@ AssertXml.Equal(..., options =>
     // Sets the maximum characters of the expected and actual inputs that should be written to the test output.
     // Default: 500 characters.
     options.MaxInputCharacters = 1000;
+
+    // Sets the format in which the different input documents will be shown in the failure report.
+    // Useful for documents that either expand in length or width to see the difference more clearly.
+    // Default: Horizontal
+    options.ReportFormat = ReportFormat.Vertical;
+
+    // Sets position in the input document that should be included in the failure report.
+    // Either Limited to the element, tag, value... that differs - only a portion of the input document will be shown where the difference resides.
+    // Useful for bigger documents where it would be hard to see the difference in the full document.
+    // Or Complete to include the entire input document in the failure report.
+    // Default: Limited
+    options.ReportScope = ReportScope.Complete;
```

Comment:
Made it configurable so you can still copy-paste it in case the XSLT transformation and development workflow requires it 👍